modules/programs/kitty.nix

```diff
@@ -1,9 +1,6 @@
 { config, lib, pkgs, ... }:
-
```

Comment:
I'm not sure why these formatting changes were made. I ran the format script at the root of the repo. I can clean this up if we want to omit the formatting changes.